client/src/components/PeerReviewDetails/PeerReviewDetails.js

```diff
@@ -124,60 +116,59 @@ const PeerReviewDetails = props => {
         </CardHeader>
         <CardBody>
           <Grid container>
-            <Grid xs={0} sm={0} md={2} item></Grid>
-            <Grid xs={12} sm={12} md={8} item>
+            <Grid xs={12} sm={12} md={12} item>
               <Table
                 tableHeaderColor="gray"
                 tableHead={peerReviewDetailHeader}
                 tableData={tempArray || null}
                 showLink={false}
               />
-              <Grid xs={6} sm={6} md={6} className={classes.grid} item>
-                <FormControl className={classes.formControl}>
-                  <InputLabel htmlFor="SelectStatus">Change Status</InputLabel>
-                  <Select
-                    value={selectedStatus}
-                    onChange={changeHandler}
-                    inputProps={{
-                      name: 'SelectStatus',
-                      id: 'SelectStatus'
-                    }}
-                  >
-                    <MenuItem className={classes.hoverEffect} value="Done">
-                      Done
-                    </MenuItem>
-                    <MenuItem
-                      className={classes.hoverEffect}
-                      value="InProgress"
-                    >
-                      In Progress
-                    </MenuItem>
-                  </Select>
-                </FormControl>
-              </Grid>
+              {ClickHandler ? (
```

Comment:
What is `ClickHandler`?

Reply:
i am reusing this component , so for some places i did not need iframe and update buttons so i am checking with click handler

Reply:
rename it as per its purpose. with current name purpose is not clear. @RakshandaWankhede